# Population control

# desired action: "pause", "resume" or "restart"
string action

---
bool success